src/app/parcels/ParcelsPage.tsx

```diff
@@ -358,6 +359,26 @@ const ParcelsPage: React.FC<{}> = () => {
         }
     }, [parcelId]);
 
+    useEffect(() => {
+        const fetchAndSetClientIdForSelectedParcel = async (): Promise<void> => {
+            const { data, error } = await supabase
+                .from("parcels")
+                .select("client_id")
+                .eq("primary_key", parcelId)
+                .single();
+
+            if (error) {
+                void logError(`Error fetching clientId from database: ${error.message}`);
```

Comment:
You could do this do log `{ error }` as metadata. (The second argument logError is metadata) 
NB `{ error }` is equivalent to `{ error: error }`
```suggestion
                void logError(`Error fetching clientId from database`, { error });
```

Could we rebase or merge dev to your branch please? I think `logError` is called something else now so you'll probs need to rename

Reply:
Changed and merged